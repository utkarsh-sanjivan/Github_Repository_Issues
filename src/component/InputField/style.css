.issues-search-field {
    width: 600px;
    margin-bottom: 5px;
}
