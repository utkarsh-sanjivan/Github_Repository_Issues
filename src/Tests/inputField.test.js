import React from 'react';
import { create } from "react-test-renderer";
import InputField from '../component/InputField/index';

test('Test Input Field Component', () => {
  const component = create(
    <InputField 
      loading={false}
      onPressEnter={event => {}}
      onChange={event => { }}
      value={'Test Text'}
    />
  );
  expect(component.toJSON()).toMatchSnapshot()
});