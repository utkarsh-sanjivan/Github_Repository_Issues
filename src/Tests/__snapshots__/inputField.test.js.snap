// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Input Field Component 1`] = `
<span
  className="ant-input-search issues-search-field ant-input-search-enter-button ant-input-group-wrapper"
>
  <span
    className="ant-input-wrapper ant-input-group"
  >
    <input
      className="ant-input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      placeholder="username/repo"
      style={null}
      type="text"
      value="Test Text"
    />
    <span
      className="ant-input-group-addon"
    >
      <button
        className="ant-btn ant-input-search-button ant-btn-primary"
        onClick={[Function]}
        onMouseDown={[Function]}
        type="button"
      >
        <span>
          Search
        </span>
      </button>
    </span>
  </span>
</span>
`;
